public/index.html

```diff
@@ -54,6 +54,17 @@
 			</figure>
 		</section>
 
-		<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+		<footer>
+			<a href="https://doc.openfisca.fr" class="cta">Get started  🙌</a>
+			<ul>
+				<li><a href="#">Home</a></li>
+				<li><a id="github">⚙ Contribute to this page</a></li>
+			</ul>
+		</footer>
+		<script type="text/javascript">
+			const filename = window.location.pathname.split("/").pop()
+			const thisOnGithub = "https://github.com/openfisca/openfisca.org/edit/master/public/" + filename
+			document.getElementById('github').href = thisOnGithub
```

Comment:
As href is the only attribute here, is it necessary to define the whole <a> in the script ? Could we define only the href string and insert it in a <a href=thisOnGithub> ?